Facebook-AutoLogin/README.md

```diff
@@ -0,0 +1,34 @@
+# Facebook-AutoLogin
+</br>
+
+## About This Project
+This is a python script that automates the facebook login process
+</br>
```

Comment:
Not needed

Reply:
Just removed project name and about this project , will that do ?

Reply:
Remove the br tag

Reply:
Oh ok  , removed the tag. Anything else?